tasks/windows.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Install MSI package
+  ansible.windows.win_package:
+    path: "{{ windows_package_url }}"
+    product_id: D67CCA1A-6708-4096-8BDE-5069739FB861
```

Comment:
This product ID is different for all versions, I think we can just remove this? 
It says it's ignored when path is a not exe and is a local accesible file path.

https://docs.ansible.com/ansible/latest/collections/ansible/windows/win_package_module.html#parameter-product_id

Reply:
Actually, this is a URL, not sure what the implications of this is.

Reply:
Cool call. I gave it a spin and it works great without product_id.